remote-access/ip-address.md

```diff
@@ -79,6 +79,68 @@ Nmap done: 256 IP addresses (4 hosts up) scanned in 2.41 seconds
 
 Here you can see a device with hostname `raspberrypi` has IP address `192.168.1.8`.
 
+### Getting IPv6 addresses by pinging from a second device (linux device or Pi)
+
+First find your own IP address(es), in other words the one of the computer you're using to find your Pi's IP address
+by `hostname -I`
+
+`fd00::ba27:ebff:feb6:f293 2001:db8:494:9d01:ba27:ebff:feb6:f293`
+
+The example shows two IP addresses. The first one is a so called unique local unicast address(`fc00::/7`). The second one is the global unicast address(`2000::/3`). It is also possible to see only one them depending on your network (router) configuration. Both addresses are valid for reaching the Raspberry Pi within your LAN. The address out of `2000::/3` is accessible world wide, provided your router's firewall is opened.
+
+
+Now use one of IPs from the first step for ping all local nodes:
+
+```
+ping -c 2 -I 2001:db8:494:9d01:ba27:ebff:feb6:f293  ff02::1`
+ping -c 2 -I 2001:db8:494:9d01:ba27:ebff:feb6:f293  ff02::1%eth0`
+```
+`-c 2` stands for sending two echo requests
+
+`-I` with the IP address, it sets the interface and the source address of the echo request,
+     it is necessary to choose the interface's IP address, 
+     `eth0` isn't sufficient - the answer would be the local link address(`fe80::/10`), we need the global or local unicast address
+     
+`ff02::1` is a well known multicast address for all nodes on the link, so it behaves like a local broadcast, usually it is defined in `/etc/hosts` so you can also use the name (`ip6-allnodes` or `ipv6-allnodes`) instead of the literal address
+
+some newer OS expect the interface ID here behind the multicast address    
```

Comment:
Capital S

Reply:
May be it is better to avoid the abbreviation of OS. (https://en.wikipedia.org/wiki/Operating_system ).
should be replaced by linux operating system or just systems